presto-benchto-benchmarks/pom.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>presto-root</artifactId>
+        <groupId>com.facebook.presto</groupId>
+        <version>0.170-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>presto-benchto-benchmarks</artifactId>
+    <name>presto-benchto-benchmarks</name>
+
+    <properties>
+        <air.main.basedir>${project.parent.basedir}</air.main.basedir>
+        <dep.logback.version>1.1.3</dep.logback.version>
+        <air.check.fail-dependency>false</air.check.fail-dependency>
+        <air.check.fail-duplicate-finder>false</air.check.fail-duplicate-finder>
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.facebook.presto</groupId>
+            <artifactId>presto-jdbc</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>com.teradata.benchto</groupId>
+            <artifactId>benchto-driver</artifactId>
+            <version>0.2-SNAPSHOT</version>
```

Comment:
It's fine to put it up for review with a snapshot, but we should update to a release version before merging.

(there are often changes needed as the result of the review, so having a snapshot during review is helpful)